DOCKER.md

```diff
@@ -40,27 +54,35 @@ Docker Compose and the project `docker-compose.yml` file will be needed to
 run functional tests which involve communicating with data systems such as
 PostgreSQL and Redis.
 
-To only run the unit tests using `Dockerfile` by itself, first change
+To run the unit tests using `Dockerfile` by itself, first change
 directories to the root of the project, then build an image, and
 finally run a container from the image:
 
 ```shell
 $ cd /path/to/project/git/clone
-$ docker build -t newrelic/newrelic-ruby-agent-tester .
-$ docker run -it newrelic/newrelic-ruby-agent-tester
+$ docker build -t rpm .
+$ docker run --rm rpm
 ```
 
-* `build -t <TAG>` applies a tag to the image during building
+The `Dockerfile` specifies a default Ruby version to test with. To override this
+version, pass the `ruby_version` build arg like so when building the image:
+
+```shell
+docker build --build-arg ruby_version=2.7 .
+```
+
+**Legend:**
+* `build -t <TAG>` applies a tag to the image during building.
 * `.` indicates "here" and tells Docker that the `Dockerfile` file can be found
   in the current directory
-* `run -it` tells Docker to run interactively (`i`) and with a pseudo TTY (`t`)
+* `run --rm` tells Docker to remove the container after the tests complete.
 
```

Comment:
Could you add an entry to the legend for `--build-arg` for completeness? I think that's the only switch that's missing

Reply:
Great idea. Implemented with [ca0c5e3](https://github.com/newrelic/newrelic-ruby-agent/pull/963/commits/ca0c5e337b2b032cf1dfdbe84a924fe23a89ba35)